src/components/app/service.ts

```diff
@@ -304,6 +304,11 @@ export function refreshGitMaterial(gitMaterialId: string) {
     })
 }
 
+export function fetchGitMaterialByCommitHash(gitMaterialId: string, commitHash: string) {
+    const URL = `${Routes.MATERIAL_BY_COMMIT_HASH}/${gitMaterialId}/${commitHash}`
```

Comment:
`MATERIAL_BY_COMMIT_HASH` is not related to commit-info. Can we change this to `MATERIAL_BY_COMMIT_INFO`

Reply:
Done